R/appStatic.R

```diff
@@ -22,6 +25,12 @@ exportStaticApp <- function(result = emptySummarisedResult(),
   omopgenerics::assertLogical(open, length = 1)
   omopgenerics::assertCharacter(logo, length = 1, null = TRUE)
   omopgenerics::assertCharacter(title, length = 1)
+  omopgenerics::assertLogical(summary, length = 1)
+  if (summary) {
```

Comment:
can we have this in an internal function? Then if we expand is easier to deal with it?